win_images/win_packaging.ps1

```diff
@@ -14,14 +14,15 @@ retryInstall git archiver psexec golang mingw StrawberryPerl; Check-Exit
 # Update service is required for dotnet
 Set-Service -Name wuauserv -StartupType "Manual"; Check-Exit
 
-# dotnet is required for wixtoolset
-# Allowing chocolaty to install dotnet breaks in an entirely
-# non-debuggable way.  Workaround this by installing it as
-# a server-feature first.
-Install-WindowsFeature -Name Net-Framework-Core; Check-Exit
-
-# Install wixtoolset for installer build & test.
-retryInstall wixtoolset; Check-Exit
+# Install dotnet
```

Comment:
This is an "add one to X" comment. If you need to repush, could you please use comments to explain the why, not the what? Even something like the previous comment ("needed for wixtoolset") would be better. Remember that this code may need to be maintained some day by non-Windows-experts. They might be reassured by a short sentence or two describing what these things are and why they're needed.

Reply:
will do

Reply:
@edsantiago addressed this in the latest update. PTAL.